packages/amplify-console-integration/projects-templates/create-react-app-auth-amplify/README.md

```diff
@@ -1,4 +1,4 @@
-# Create-react-app with AWS Amplify Auth 
+# Create-react-app with AWS Amplify Auth
```

Comment:
Same with this file

Reply:
I think it's good to keep this read me file, so readers know that the auth starter sample project is used in the console hosting e2e test.